.github/workflows/main.yml

```diff
@@ -99,6 +99,7 @@ jobs:
         CARGO_TARGET_DIR: target
     - run: cargo test -p cargo-platform
     - run: cargo test -p cargo-util
+    - run: cargo test -p home
```

Comment:
I don't think this line is doing what the intended effect is. This is testing `home` from `crates-io`, not the one in this repository. The reason the other two work with `-p` is because they are path-dependencies. 

I think the fix here is to use something like `--manifest-path crates/home/Cargo.toml`

Reply:
Good point, made that change in https://github.com/rust-lang/cargo/pull/11359/commits/18a06cc809cd0c7dc2a7297ad4a2b4b98b829d22